### PR TITLE
Python3 compatibility

### PR DESCRIPTION
**Pull request by t-ionut**

### CODE REVIEW

**Comment by bogdanpetrea** [comment]
Maybe add parantheses to print calls in project: https://github.com/PressLabs/zipa/search?utf8=%E2%9C%93&q=print

**Comment by t-ionut** [comment]
Good catch, I forgot about the examlpes :)

File: zipa/module.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    self = sys.modules['zipa']
+    sys.modules[name] = SelfWrapper(self)
+
+
+class ModuleImporter(object):
+    def __init__(self, module_name):
+        self.name = module_name
+
+    def find_module(self, name, path=None):
+        if name.startswith('{}.'.format(self.name)):
```
Comment (krodyrobi) [inline zipa/module.py]: maybe it is worth trying __name__ here instead? dunno if it fits but it would easier to call the importer etc.

**Comment by krodyrobi** [comment]
Apart from what I've said LGTM. @vtemian can you have a look :P

File: zipa/module.py (hunk 1 above)
Comment (krodyrobi) [inline zipa/module.py]: Nvm saw it propagated.

File: zipa/module.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import sys
+
+from .magic import SelfWrapper
+
+
+def register_module(name):
+    self = sys.modules['zipa']
```
Comment (vtemian) [inline zipa/module.py]: You can replace `self` with `zipa`

**Comment by vtemian** [comment]
Can you also change `.travis.yml` file and add `3.4` and `3.5`?

**Pull request merged**